src/MusicalScore/ScoreIO/MusicSymbolModules/ChordSymbolReader.ts

```diff
@@ -138,19 +139,28 @@ export class ChordSymbolReader {
                 return undefined;
             }
 
-            let text: ChordDegreeText;
-            try {
-                text = ChordDegreeText[degreeType.value.trim().toLowerCase()];
-            } catch (ex) {
-                const errorMsg: string = ITextTranslation.translateText("ReaderErrorMessages/ChordSymbolError",
-                                                                        "Invalid chord symbol");
-                musicSheet.SheetErrors.pushMeasureError(errorMsg);
-                log.debug("InstrumentReader.readChordSymbol", errorMsg, ex);
-                return undefined;
-            }
+                let text: ChordDegreeText;
+                try {
+                    text = ChordDegreeText[degreeType.value.trim().toLowerCase()];
+                } catch (ex) {
+                    const errorMsg: string = ITextTranslation.translateText(
+                        "ReaderErrorMessages/ChordSymbolError",
+                        "Invalid chord symbol"
+                    );
+                    musicSheet.SheetErrors.pushMeasureError(errorMsg);
+                    log.debug("InstrumentReader.readChordSymbol", errorMsg, ex);
+                    return undefined;
+                }
```

Comment:
wrong indentation: reduce indentation by one block/level

Reply:
Actually, it looks like I had the indentation above wrong...